app/controllers/users_controller.rb

```diff
@@ -1,3 +1,5 @@
 class UsersController < ApplicationController
-  def hello; end
+  def index; end
+
+  def show; end
```

Comment:
- [ ] You are missing the accompanying user show as well: `app/views/users/show.html.erb`. Kindly add that file.

Reply:
This issue is well resolved! ✔️